### PR TITLE
Fixed #5182 - All warnings in "Internationalization"

### LINKED ISSUE #5182: All warnings in "Internationalization" and "Internationalization:Bidirectional Text" category
## Description 
 We would like to fix all the remaining lint issues on Oppia Android in "Internationalization" category.

## Steps To Reproduce
Run the following command from a terminal in a checked-out Oppia Android repo:

`./gradlew :app:lint`

Output:  All warnings in "Internationalization" category

## Expected Behavior
Running the command above should not show any errors.

### PR DESCRIPTION
**Pull request by Rd4dev**
## Explanation

Fix #5182 [part of #5169]

Fixes All warnings in "Internationalization" caused by
- hardcoded text : Fixed by assigning `string resource` values
- padding/margins : Fixed by setting `padding end` values

## ScreenShot:
Lint Issues Internationalization Before After

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

<br>

### CODE REVIEW

File: app/src/main/res/layout-sw600dp/administrator_controls_learner_analytics_view.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
       android:paddingTop="28dp"
       android:paddingEnd="16dp"
       android:paddingBottom="20dp"
-      android:text="Learner Study Analytics"
+      android:text="@string/learner_analytics_title"
```
Comment (adhiamboperes) [inline app/src/main/res/layout-sw600dp/administrator_controls_learner_analytics_view.xml]: This is a label and not a title

File: app/src/main/res/layout/activity_input_interaction_view_test.xml [hunk 2]
```diff
@@ -122,7 +122,7 @@
         android:layout_margin="8dp"
         android:background="@drawable/edit_text_background"
         android:focusable="true"
-        android:hint="Write the digit here."
+        android:hint="@string/text_number_input_hint"
```
Comment (adhiamboperes) [inline app/src/main/res/layout/activity_input_interaction_view_test.xml]:
```suggestion
        android:hint="@string/test_number_input_interaction_hint"
```

File: app/src/main/res/layout/activity_input_interaction_view_test.xml [hunk 3]
```diff
@@ -154,7 +154,7 @@
         android:layout_margin="8dp"
         android:background="@drawable/edit_text_background"
         android:focusable="true"
-        android:hint="Write here."
+        android:hint="@string/text_text_input_hint"
```
Comment (adhiamboperes) [inline app/src/main/res/layout/activity_input_interaction_view_test.xml]:
```suggestion
        android:hint="@string/test_text_input_interaction_hint"
```

File: app/src/main/res/layout/activity_input_interaction_view_test.xml [hunk 4]
```diff
@@ -181,7 +181,7 @@
         android:layout_height="wrap_content"
         android:layout_margin="8dp"
         android:onClick="getPendingAnswerErrorOnSubmitClick"
-        android:text="Submit"
+        android:text="@string/state_submit_button"
```
Comment (adhiamboperes) [inline app/src/main/res/layout/activity_input_interaction_view_test.xml]:
```suggestion
        android:text="@string/submit_button_text"
```

File: app/src/main/res/layout/spotlight_fragment_test_activity.xml [hunk 5]
```diff
@@ -15,7 +15,7 @@
       android:id="@+id/sample_spotlight_target"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:text="This is a sample text view"
+      android:text="@string/sample_text_view"
```
Comment (adhiamboperes) [inline app/src/main/res/layout/spotlight_fragment_test_activity.xml]:
```suggestion
      android:text="@string/sample_text_view_text"
```

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @Rd4dev!

I have requested changes inline.

File: app/src/main/res/layout/spotlight_fragment_test_activity.xml (hunk 5 above)
Comment (Rd4dev) [reply app/src/main/res/layout/spotlight_fragment_test_activity.xml]: Renamed the string resource 'sample_text_view' to 'sample_text_view_text'

File: app/src/main/res/layout/activity_input_interaction_view_test.xml (hunk 4 above)
Comment (Rd4dev) [reply app/src/main/res/layout/activity_input_interaction_view_test.xml]: In the past, the existing 'state_submit_button' was utilized, but a new string resource named 'submit_button_text' has now been created for the activity input interaction's submit button

File: app/src/main/res/layout/activity_input_interaction_view_test.xml (hunk 3 above)
Comment (Rd4dev) [reply app/src/main/res/layout/activity_input_interaction_view_test.xml]: Renamed the string resource 'text_text_input_hint' to 'test_text_input_interaction_hint'

File: app/src/main/res/layout/activity_input_interaction_view_test.xml (hunk 2 above)
Comment (Rd4dev) [reply app/src/main/res/layout/activity_input_interaction_view_test.xml]: Renamed the string resource 'text_number_input_hint' to 'test_number_input_interaction_hint'

File: app/src/main/res/layout-sw600dp/administrator_controls_learner_analytics_view.xml (hunk 1 above)
Comment (Rd4dev) [reply app/src/main/res/layout-sw600dp/administrator_controls_learner_analytics_view.xml]: For the label - Renamed the string resource 'learner_analytics_title' to 'learner_analytics_label_text'

**Comment by Rd4dev** [comment]
Thank you for the review @adhiamboperes. I have incorporated the requested changes!

**Review by adhiamboperes — APPROVED** [review]
Thanks @Rd4dev. This LGTM.

**Pull request merged**